src/biome/text/loggers.py

```diff
@@ -179,3 +188,67 @@ def end_train(self, results: TrainingResults):
             ]
         finally:
             self._client.set_terminated(self._run_id)
+
+
+class WandBLogger(BaseTrainLogger):
+    """Logger for WandB
+
+    Parameters
+    ----------
+    project_name
+        Name of your WandB project
+    run_name
+        Name of your run
+    tags
+        Extra arguments used as tags to created experiment run
+    """
+
+    def __init__(self, project_name: str = "biome", run_name: str = None, tags: List[str] = None):
+        if wandb.api.api_key is None:
+            wandb.termwarn("W&B installed but not logged in. "
+                           "Run `wandb login` or `import wandb; wandb.login()` or set the WANDB_API_KEY env variable.")
+
+        self.project_name = project_name
+        self.run_name = run_name
+        self.tags = tags
+
+    def init_train(
+        self,
+        pipeline: "Pipeline",
+        trainer_configuration: "TrainerConfiguration",
+        training: InstancesDataset,
+        validation: Optional[InstancesDataset] = None,
+        test: Optional[InstancesDataset] = None,
+    ):
+        config = {
+            "pipeline": pipeline.config.as_dict(),
+            "trainer": dataclasses.asdict(trainer_configuration),
```

Comment:
Could we log here the `num_parameters`, it is an important factor for choosing/comparing models.

Reply:
yes good point. Done